### ISSUE TITLE
建议在标注模块（浮动面板及标注模式）中加入类似Remarkerbe扩展的功能

### ISSUE THREAD

**Issue by taijiwolf** [Kenshin/simpread#3009]
这是一个比较古老的chrome扩展[Remarkerbe](https://chrome.google.com/webstore/detail/remarkerbe/aklkmkchojnnjbmggchdeagdgcflfhnc)，作为一款标注高亮工具，其特点在于，它会将高亮的文字按照特定的规则重新排版（类似于某种拆书方式）。对于阅读中短篇幅的知识型文章辅助性很高。可惜很早就已经不再更新，保存功能也已失效，多年来，一直未找到类似的代替工具。在使用简悦的过程中，觉得此功能与简悦阅读模式中的标注功能很契合，在稍后读中使用极为便利。因此冒昧提出建议。
下方图片是一个使用例子：
![屏幕截图 2021-11-05 113826](https://user-images.githubusercontent.com/76593856/140457235-ed811702-8732-4d78-a101-0456095b42bc.png)
![屏幕截图 2021-11-05 113912](https://user-images.githubusercontent.com/76593856/140457267-f90e30d1-6a50-4b9f-8cb6-e7114fd8c99b.png)
![屏幕截图 2021-11-05 113936](https://user-images.githubusercontent.com/76593856/140457279-579a0771-f577-4ae5-8fec-e161400ced7f.png)

**Comment by Kenshin** [comment]
嗨，谢谢建议

我有研究过 Remarkerbe 简悦的备注设计方案设计也参考于它

![image](https://user-images.githubusercontent.com/81074/140457723-5c491cf3-b235-42ce-9f5f-1ce4f9f4f8bd.png)

你的意思是在标注的颜色和样式基础上人为增加一些标注的属性？

**Comment by taijiwolf** [comment]
是的。其实Remarkerbe本质就是标注。他的特点是各个标注带有属性，如主题（topic）,是什么（what）,为什么（why）,怎么做（how）以及其他，且各个标注之前形成层级逻辑关系。简悦的标注如果功能深化一下我觉得完全可以做到。而且简悦的标注拖曳功能对于层级调整也很方便。（Remarkerbe就不行，标注过程中顺序错了再调整就很麻烦）

**Comment by Kenshin** [comment]
okay 收到，谢谢建议，我考虑下。

**Comment by Kenshin** [comment]
嗨，这个 Feature 已记录，请告诉我你的邮箱地址（发到 help@simpread.pro 即可），此功能发布后会通知你。

因为要整理一些 Issues ，所以此 Issues 会被删除掉。

PS：如果可以进 Telegram 群的话，也请进群联络我。

🙏
